app/views/v2/pkg/show.mustache

```diff
@@ -124,11 +124,6 @@ import 'package:{{package}}/{{library}}';
       <a class="link" href="{{package.documentation}}">{{package.nice_documentation}}</a>
     {{/package.documentation}}
 
-    {{#package.crossdart}}
-      <h3 class="title">Source code (hyperlinked)</h3>
-      <a class="link" href="{{package.crossdart}}">{{package.nice_crossdart}}</a>
```

Comment:
Should cleanup the `package.*crossdart` values, too!

Reply:
The model references? I assume those are stilled used by the v1 code?

Reply:
We should kill the feature there, too – while you're at it 😄

Reply:
Done!